pkg/signature/jws.go

```diff
@@ -52,7 +53,10 @@ func NewVerifierFromFiles(certPaths []string) (*jws.Verifier, error) {
 			return nil, err
 		}
 		if !verifier.VerifyOptions.Roots.AppendCertsFromPEM(data) {
-			return nil, fmt.Errorf("failed to parse PEM certificate: %q", path)
+			certPEM := pem.EncodeToMemory(&pem.Block{Type: "CERTIFICATE", Bytes: data})
```

Comment:
Probably we can add a comment to explain that we're trying to convert DER to PEM? But it looks good to me overall.

Reply:
> Probably we can add a comment to explain that we're trying to convert DER to PEM? But it looks good to me overall.

Sure. Will add a comment to it.

Reply:
Did you test it with some corner cases? like empty file, wrong cert file.

Reply:
> Did you test it with some corner cases? like empty file, wrong cert file.

Yes, both scenarios are covered. Also tested end to end with the example net-monitor image.